src/Magento/FunctionalTestingFramework/Suite/SuiteGenerator.php

```diff
@@ -178,7 +178,7 @@ private function validateTestsReferencedInSuite($suiteName, $testsReferenced, $o
     {
         $suiteRef = $originalSuiteName ?? $suiteName;
         $possibleTestRef = SuiteObjectHandler::getInstance()->getObject($suiteRef)->getTests();
-        $errorMsg = "Cannot reference tests whcih are not declared as part of suite.";
+        $errorMsg = "Cannot reference tests which are not declared as part of suite.";
```

Comment:
Can we display $testsReferenced in the error message?

Reply:
I believe so, I'll try to add this.

Reply:
Fixed